### DISCUSSION TITLE
Welcome to Chainlink Local Discussions!

### DISCUSSION THREAD

**Discussion by andrejrakic** [smartcontractkit/chainlink-local#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

And don't forget the useful links to get started with Chainlink Local
- [Chainlink Local Documentation](https://cll-devrel.gitbook.io/chainlink-local-documentation/)
- [Chainlink Local Examples YouTube Playlist](https://www.youtube.com/watch?v=rEVjU9tOf74&list=PL3ZUTf1nxlFyHKswTYFa2tffUsR94KAEv)
- [Chainlink CCIP Documentation](https://docs.chain.link/ccip/)

**Comment by EricVonza** [comment]
Hello everyone . I'm new to Chainlink ,my firm specializes in working with B2B app transactions ,aggregating realtime pricing from various currencies. We're interested in contributing to decentralization by offering these services within the Chainlink ecosystem, any assistance ;)
